n_t_with_s/README.md

```diff
@@ -0,0 +1,170 @@
+[smc++](https://github.com/popgenmethods/smcpp).
+There are four target rules that can be executed with the given parameters: 
+`compound_msmc`,
+`compound_smcpp`,
+`compound_stairwayplot`, 
+or you can run all three on the same simulated data with rule `all`.
+
+To run an analysis, create a directory (wherever you want)
+where all results, and intermediate
+files will be stored. Next, create and place a file named `config.json` in it.
+The json file must contain key : value combos described below. An example 
+might look like this:
+
+```json
+{
+    "seed" : 12345,
+    "population_id" : 0,
+    "num_samples_per_population" : [20, 0, 0],
+    "num_sampled_genomes_msmc" : "2,8",
+    "num_sampled_genomes_per_replicate": "2,8",
+    "num_msmc_iterations" : 20,
+    "replicates" : 1, 
+    "species" : "HomSap",
+    "model" : "OutOfAfricaArchaicAdmixture_5R19",
+    "genetic_map" : "HapMapII_GRCh37",
+    "chrm_list" : "chr22",
+    "mask_file" : "masks/HapmapII_GRCh37.mask.bed",
+    "dfe_id": "Gamma_K17",
+    "selection":"all", 
+    "annotation_id": "ensembl_havana_104_exons"
+}
+```
+
+Once you have created a directory which contains the config file
+simply run snakemake from _this_ directory (n_t), and point it to your analysis run
+directory, like so
+
+`snakemake -j 40 --config config="/projects/kernlab/jgallowa/homo_sapiens_Gutenkunst_0"`
+
+where `-j` is the number cores available to run jobs in parallel, and 
+`--config` points to the _directory_ that contains the config file.
+
+### Cluster environments
+Our workflow can also be run on a cluster. To do so, it requires
+the setup of a `.json` configuration file that lets `snakemake`
+know about your cluster. We have provided an example of 
+such a file in `cluster_talapas.json` that is for use with a
+University of Oregon SLURM cluster. 
+
+```json
+{
+    "__default__" :
+    {
+        "time" : "02:30:00",
+        "n" : 1,
+        "partition" : "kern,preempt",
+        "mem" : "16G",
+        "cores" : "4",
+    },
+    "run_msmc" :
+    {
+        "time" : "05:00:00",
+        "mem" : "32G",
+        "cores" : "4",
+    }
+    
+}
+```
+
+At a minimum, you should
+edit the names of the partition to match those on your own HPC.
+The workflow can then be launched with the call
+
+`snakemake -j 999 --config config="/projects/kernlab/jgallowa/homo_sapiens_Gutenkunst_0" --cluster-config cluster_talapas.json --cluster "sbatch -p {cluster.partition} -n {cluster.n} -t {cluster.time} --mem-per-cpu {cluster.mem} -c {cluster.cores}"`
+
+(it may prove useful to simply put this command in a bash script)
+
+and jobs will be automatically farmed out to the cluster
+
+### Final output
+The current final output is a plot comparing stairwayplot and smc++ estimates of $N(t)$, i.e., 
+`homo_sapiens_Gutenkunst/all_estimated_Ne_{selection}.png` , 
+with selection being 'none', 'all' or applied to the 'exons'.
+
+
+## Parameter Description
```

Comment:
might be a little more specific about where this params are? e.g., 
```suggestion
## Config file Parameter Description
```

Reply:
added change